test/unit/asrouter/CFRPageActions.test.js

```diff
@@ -468,12 +471,12 @@ describe("CFRPageActions", () => {
         });
       });
       it("should set the main action correctly", async () => {
+        sinon.stub(pageAction, "_fetchLatestAddonVersion").returns(Promise.resolve("latest-addon.xpi"));
```

Comment:
you can actually do `sinon.stub(pageAction, "_fetchLatestAddonVersion").resolves("latest-addon.xpi")` here